src/Catel.Core/Catel.Core.Shared/Runtime/Serialization/SerializerBase.general.cs

```diff
@@ -708,6 +708,51 @@ protected virtual bool ShouldSerializeUsingParseAndToString(MemberValue memberVa
             });
         }
 
+
+        /// <summary>
+        /// Returns whether the enum member value should be serialized as string.
+        /// </summary>
+        /// <param name="memberValue"></param>
+        /// <param name="checkActualMemberType"></param>
+        /// <returns></returns>
+        protected virtual bool ShouldSerializeUsingEnumAsString(MemberValue memberValue, bool checkActualMemberType)
+        {
+            var cacheKey = $"{memberValue.ModelTypeName}|{memberValue.MemberTypeName}|{checkActualMemberType}";
+
+            return _shouldSerializeUsingParseCache.GetFromCacheOrFetch(cacheKey, () =>
```

Comment:
1. Ok sorry my mistake
2. Ok but in the method above UsingEnumToString and EnumAsString is also mixed. So every method attribute field and so on should be called EnumAsString except "ShouldSerializeEnumMemberUsingToString" is this correct?